templates/snippets/head_meta.html

```diff
@@ -1,20 +1,20 @@
 {% load static cdh_tags %}
 {# html metadata #}
-    <meta name="keywords" content="{% block meta_keywords %}{{ page.keywords.all|join:', ' }}{% endblock %}"/>
-    <meta name="description" content="{% block meta_description %}{{ page.description }}{% endblock %}"/>
-    {# determine preview image: specified by page, cdh icon based on url, or default image #}
-    {% firstof preview_image page.get_absolute_url|url_to_icon_path default_preview_image as meta_preview_image %}
-    {# open graph metadata #}
-    <meta property="og:title" content="{{ page.meta_title }}" />
-    <meta property="og:type" content="{{ opengraph_type|default:'website' }}" />
-    <meta property="og:url" content="{{ request.build_absolute_uri }}" />
-    <meta property="og:image" content="{{ meta_preview_image }}" />
-    <meta property="og:description" content="{{ page.description }}"/>
-    <meta property="og:site_name" content="Center for Digital Humanities @ Princeton University"/>
-    {# twitter card #}
-    <meta name="twitter:card" content="{{ twitter_card_type|default:'summary' }}" />
-    <meta name="twitter:site" content="@PrincetonDH" />
-    <meta name="twitter:title" content="{{ page.meta_title }}"/>
-    <meta name="twitter:description" content="{{ page.description }}" />
-    <meta name="twitter:image" content="{% firstof twitter_image meta_preview_image %}" />
+<meta name="keywords" content="{{ page.tags.all|join:', ' }}"/>
+<meta name="description" content="{{ page.search_description }}"/>
+{# determine preview image: specified by page, cdh icon based on url, or default image #}
+{% firstof preview_image page.get_url|url_to_icon_path default_preview_image as meta_preview_image %}
+{# open graph metadata #}
+<meta property="og:title" content="{{ page.seo_title }}" />
```

Comment:
I checked a little bit, title is empty for all the pages I looked at. I think this needs to be something like `{% firstof page.seo_title page.title page_title %}` — I checked, that's the context variable set in non-page views like the project list pages. Maybe set this once similar to the preview image.

Reply:
I _think_ seo_title is always set (it was set every time I checked), but that order looks right regardless. thanks!